--- tenshi-0.3.4/tenshi	2005-06-16 12:03:21.000000000 +0300
+++ /home/glen/tenshi	2006-05-09 18:45:56.926641341 +0300
@@ -453,7 +453,7 @@
     $debug && debug(11,$queue);
  
     my $subject = $main{$queue}{'subject'} || $subject;
-    my $smtp = Net::SMTP->new($mailserver, Timeout => 10);
+    my $smtp = Net::SMTP->new($mailserver, Timeout => 10, Debug => $debug);
         
     if (!$smtp) { 
         print "[ERROR] could not contact $mailserver:25\n"; 
